daemon/algod/server.go

```diff
@@ -78,7 +79,14 @@ func (s *Server) Initialize(cfg config.Local, phonebookAddresses []string, genes
 			maxLogAge = 0
 		}
 	}
-	logWriter := logging.MakeCyclicFileWriter(liveLog, archive, cfg.LogSizeLimit, maxLogAge)
+
+	var logWriter io.Writer
+	if cfg.LogFileName == "-" {
```

Comment:
@tsachiherman where is a good place to test this logic? server_test.go, could be a place but I did not find any tests on logging in there. Perhaps higher up the test pyramid?

Reply:
I agree that server_test.go isn't the best place. It's more of a UI feature, as can be tested using expect testing.
We already have a test for algod in `./go-algorand/test/e2e-go/cli/algod/expect`. You can take `algodTelemetryLocationTest.exp` as a starting point and try to create another test to verify your change.
Using expect the first time might not be very trivial, but it's getting straight forward once you get used to it.

( as an alternative, I'm also ok if you'll have a go test for that by capturing the output vs. verifying the output goes to node.log file )